test/integration/framework/controlplane_utils.go

```diff
@@ -65,6 +65,13 @@ const (
 	UnprivilegedUserToken = "unprivileged-user"
 )
 
+func init() {
+	// Log at level 4 by default. Can be changed via -v on the command
+	// line. k8s.io/component-base/logs called klog.InitFlags for us.
+	v := flag.Lookup("v").Value
+	v.Set("4")
+}
```

Comment:
why? This is affecting everybody who imports. Before it was just effective if you created an apiserver.

Reply:
The "why" is that the previous approach didn't work for -v0 on the command line, which is relevant for benchmarks.

I was assuming that setting -v4 as default would be desirable for all integration tests. If that is not the case, then `startAPIServerOrDie` must allow controlling whether it raises the verbosity.

Would `controlplane.Config` be a suitable place for that?

Reply:
> Would controlplane.Config be a suitable place for that?

Hmm, probably not.

Reply:
Simpler than an explicit parameter would be a global variable in controlplane_utils.go:
```
// MinVerbosity determines the minimum klog verbosity when running tests that involve the apiserver.
// This overrides the -v value from the command line, i.e. -v0 has no effect when MinVerbosity is 4 (the default).
// Tests can opt out of this by setting MinVerbosity before starting the control plane.
var MinVerbosity = 4
```

@dims: I think you added that code originally. Any thoughts?

Reply:
I replaced the init function with the global variable.

@mtaufen : can you take another look?